ice/recipes/elicit/search.py

```diff
@@ -51,10 +52,13 @@ async def elicit_search(
 
     log.info(f"Searching Elicit for query: {question}, endpoint: {endpoint}")
 
-    filters = dict(has_pdf=has_pdf_filter)
+    all_filters = dict(has_pdf=has_pdf_filter)
+
+    if filters is not None:
+        all_filters = {**filters, **all_filters}
 
     request_body = make_request_body(
-        query=question, num_papers=num_papers, page=page, filters=filters
+        query=question, num_papers=num_papers, page=page, filters=all_filters
```

Comment:
Looks good, but I think this version is a bit clearer:

```suggestion
        query=question, num_papers=num_papers, page=page, filters=(filters or {}) | dict(has_pdf=has_pdf_filter)
```

Reply:
word

Reply:
I did not know about https://peps.python.org/pep-0584/